Submitted By: Alex Potter (alexp at apcuk dot dyndns dot org)
Date: 2004-03-11
Initial Package Version: 2.05
Origin: Alex Potter
Description: Adjust the path in the documentation Makefile to enable online 
	     help to find the required files.
	     Install the abiword man page.
 
diff -Naur abiword-2.0.5.orig/abiword-docs/Makefile abiword-2.0.5/abiword-docs/Makefile
--- abiword-2.0.5.orig/abiword-docs/Makefile	2003-09-17 17:39:13.000000000 +0000
+++ abiword-2.0.5/abiword-docs/Makefile	2004-03-11 07:26:46.000000000 +0000
@@ -1,6 +1,6 @@
 all:	docs-en-US-install
 
-prefix = /usr/local
+prefix = /usr
 
 docdir = $(prefix)/share/AbiSuite-2.0/AbiWord/help
 
@@ -20,3 +20,4 @@
 	    fi ; \
 	  done; \
 	done
+	@cp man/abiword.8 /usr/man/man8
